src/misc/ErrorHandler.js

```diff
@@ -1,23 +1,33 @@
 // @flow
 import {assertMainOrNodeBoot, isTest} from "../api/common/Env"
-import {SecondFactorPendingError} from "../api/common/error/SecondFactorPendingError"
+import {delay} from "@tutao/tutanota-utils"
 
 assertMainOrNodeBoot()
 
-export async function handleUncaughtError(e: Error) {
+function throttledImport<R>(ms: number, fn: (() => Promise<R>)): (() => Promise<R>) {
+	let lastTry = 0
+	return async () => {
+		let previousTry = lastTry
+		lastTry = Date.now()
+		if (previousTry - Date.now() < ms) {
+			await delay(previousTry - Date.now())
+		}
+		return fn()
+	}
+}
+
+const importErrorHandler = throttledImport(500, () => import("./ErrorHandlerImpl.js"))
 
+export async function handleUncaughtError(e: Error) {
 	if (isTest()) {
 		throw e
 	}
-	if (e instanceof SecondFactorPendingError) {
-		// ignore
-	} else {
-		console.log("error", e)
-	}
+
+	console.log("error", e, e.stack)
 
 	try {
 		// decoupled to remove size of boot bundle
```

Comment:
It would also be nice to make this comment more prominent as a doc comment at the top explaining why this file exists in the first place